.buildkite/test-pipeline.yaml

```diff
@@ -179,7 +179,11 @@ steps:
 
 - label: Samplers Test
   #mirror_hardwares: [amd]
-  command: pytest -v -s samplers
+  commands:
+    - pytest -v -s samplers
+    # Test for flashinfer samplers
+    - pip install https://github.com/flashinfer-ai/flashinfer/releases/download/v0.1.2/flashinfer-0.1.2+cu121torch2.4-cp310-cp310-linux_x86_64.whl
```

Comment:
@simon-mo we should be able to remove all of these since the docker image already installed right?

Reply:
Correct https://github.com/vllm-project/vllm/blob/6e4852ce28ad57dc440067778464ac61e0621899/Dockerfile#L196-L204